### ISSUE TITLE
Create new columns with the sentiment analysis, frames and keywords extraction/topic

### ISSUE THREAD

**Issue by JuanSCampos** [JuanS286/gif_auto_audio_description#6]

**Comment by JuanSCampos** [comment]
We met on Sunday, October 6th and agreed on the following actions to improve Topic Detection:

1. Test another NLP only on the Uncategorized rows (Spacy or another one) or with fewer records (Ana's file).
2. Join the Uncategorized with the Text category and leave them as Others. We can also use the previous point as an input to improve the Topic categorization further.

**Comment by JuanSCampos** [comment]
- We create the sentiment analysis column using the TextBlob library. - Ana is finishing another sentiment analysis using Power BI analytics tools to compare the results.

- We create two columns Action_1 and Action_2 identifying the verbs in the description column using the NLTK library.

- We took Norma and Timi's input .csv file from the categorization topic. We selected the one done manually since we tried several ways to use NLP libraries like Spacy without getting results due to high processing time. Additionally, we tried to run the categorization with Spacy only with the actions to reduce processing time. Still, we got many different categories, and many categories were left empty because the model could not categorize everything.
